js/interface.js

```diff
@@ -550,11 +550,22 @@ function uploadFiles(files) {
     files.forEach(function(file) {
       addFile(file);
     });
-
+    sortImageFiles();
     $progress.addClass('hidden');
   });
 }
 
+function sortImageFiles() {
+  var sort_by_name = function (a, b) {
```

Comment:
please use camelCase for function names as per code standards

Reply:
Fixed.